projects/custody-key-dates-and-delius/src/main/resources/application.yml

```diff
@@ -44,6 +44,7 @@ spring.config.activate.on-profile: [ "dev", "integration-test" ]
 spring:
   datasource.url: jdbc:h2:file:./dev;MODE=Oracle;DEFAULT_NULL_ORDERING=HIGH;AUTO_SERVER=true;AUTO_SERVER_PORT=9092
   jpa.hibernate.ddl-auto: create-drop
+  security.oauth2.resourceserver.jwt.public-key-location: classpath:local-public-key.pub
```

Comment:
You'll need to add the extra auth env vars to the values files too to make this work as an API server e.g. https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/projects/approved-premises-and-delius/deploy/values-dev.yml#L11-L12